### PR TITLE
Fix #2193 - Add new pricing copy to the VPN promo banner

### LINKED ISSUE #2193: Replace Firefox Monitor: Get Mozilla VPN banner with 4.99 pricing special banner
We are running a new campaign for Mozilla VPN, before we end our introductory pricing and need to replace the banner on Firefox Monitor.

Link to asset: https://drive.google.com/drive/folders/16ghFYfkLYPGm60ZqSacXS-eMCU5ItAXs?usp=sharing

This banner should only go live in US, UK, CA, Singapore, New Zealand, and Malaysia in English only. 

The banner should be dismissable.

UTM parameter: https://www.mozilla.org/products/vpn/?utm_source=firefox-monitor&utm_medium=firefox-monitor&utm_campaign=intro-pricing 

Asset should stay live until July 13th then be removed. I'll file another issue for changing the asset back to the existing asset.

### PR DESCRIPTION
**Pull request by maxxcrawford**
Note – These strings are intentionally not being translated. It is only for the next few weeks and will revert back to the previous text.

### CODE REVIEW

File: public/css/vpn-promo-banner.css [hunk 1]
```diff
@@ -31,8 +31,12 @@ body {
   max-height: 0;
 }
 
+.vpn-promo-banner + .recruitment-banner {
```
Comment (maxxcrawford) [inline public/css/vpn-promo-banner.css:34]: This statement only allows for one "banner" to be visible at a time.

File: public/css/vpn-promo-banner.css [hunk 2]
```diff
@@ -150,4 +160,7 @@ body {
     top: 0.25rem;
     right: 0.25rem;
   }
+  .vpn-promo-banner a {
+    flex-shrink: 0;
+  }
```
Comment (maxxcrawford) [inline public/css/vpn-promo-banner.css:165]: Added to adjust the design better for non-English languages. It allows the button copy to stay on one-line when on larger screens.

File: public/js/monitor.js [hunk 3]
```diff
@@ -528,10 +542,14 @@ function vpnBannerLogic() {
...
   // Only show banner if users first language is some English-locale variant
-  if (["en", "de", "fr"].some(lang=>preferredLanguages[0].includes(lang))) {
+  if (["de", "fr"].some(lang=>preferredLanguages[0].includes(lang))) {
     vpnBannerLogic();
   }
 
+  if (["en"].some(lang=>preferredLanguages[0].includes(lang))) {
+    vpnBannerLogic({isEnglish: true});
+  }
+
```
Comment (maxxcrawford) [inline public/js/monitor.js:552]: This logic is just reused from above. If there's a better way to do this without the `.some`, let me know.

File: views/partials/vpn-promo.hbs [hunk 4]
```diff
@@ -5,7 +5,7 @@
 			<strong>{{ getString "vpn-promo-headline" }}</strong>
 			<span>{{ getString "vpn-promo-copy" }}</span>
 		</div>
-		<a id="" href="https://www.mozilla.org/products/vpn/?utm_source=monitor&utm_medium=monitor&utm_campaign=mozillavpn-monitor-top-banner">{{ getString "vpn-promo-cta" }}</a>
+		<a class="vpn-promo-cta" href="https://www.mozilla.org/products/vpn/?utm_source=monitor&utm_medium=monitor&utm_campaign=mozillavpn-monitor-top-banner">{{ getString "vpn-promo-cta" }}</a>
```
Comment (maxxcrawford) [inline views/partials/vpn-promo.hbs:8]: Added this class for CSS targeting (and removed the empty `id`).

**Review by groovecoder — APPROVED** [review]
Code looks good. Did not spot check though.

**Pull request merged**